src/Tree.tsx

```
@@ -561,36 +562,38 @@ class Tree<TreeDataType extends DataNode | BasicDataNode = DataNode> extends Rea
+        }
 
         let newExpandedKeys = [...expandedKeys];
-        const entity = getEntity(keyEntities, node.props.eventKey);
+        const entity = getEntity(keyEntities, nodeProps.eventKey);
 
         if (entity && (entity.children || []).length) {
-          newExpandedKeys = arrAdd(expandedKeys, node.props.eventKey);
+          newExpandedKeys = arrAdd(expandedKeys, nodeProps.eventKey);
         }
 
         if (!this.props.hasOwnProperty('expandedKeys')) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)